experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -357,6 +358,30 @@ class Meta:
         dataclass = NimbusConfigurationDataClass
 
 
+@dataclass
+class NimbusFmlErrorDataClass:
+    line: int
+    col: int
+    message: str
+    highlight: str
+
+
+@dataclass
+class NimbusFmlErrorListDataClass:
+    errors: typing.List[typing.Optional[NimbusFmlErrorDataClass]]
```

Comment:
it's already imported at the top, Pylance complains when you don't say the whole `typing.List`. It's like that in like 40 other places in this file, so I'm gonna keep this as-is

Reply:
`list` not `List`. As of 3.8 you can subscript list dict etc for type annotations instead of the ones exported by typing.

Reply:
Also what I meant with the import was:

```python
from typing import Optional

# ...

errors: list[Optional[T]]

```

Reply:
Yeah, that makes sense. I'd like to keep this as-is since it's consistent with the rest of the file, though, and I'm not going to change all of the instances of this in this PR. I can file a cleanup ticket to change this 👍